dockers/docker-snmp-sv2/snmpd

```diff
@@ -8,4 +8,4 @@ export MIBS=
 SNMPDRUN=yes
 
 # snmpd options (use syslog, close stdin/out/err).
-SNMPDOPTS='-LS4d -Lf /dev/null -u snmp -g snmp -I -smux,mteTrigger,mteTriggerConf,ifTable,ifXTable -p /run/snmpd.pid'
+SNMPDOPTS='-LS4d -Lf /dev/null -u Debian-snmp -g Debian-snmp -I -smux,mteTrigger,mteTriggerConf,ifTable,ifXTable -p /run/snmpd.pid'
```

Comment:
you are probably right.

net-snmp (5.7.3+dfsg-1.1) unstable; urgency=medium

  * debian/snmpd.{preinst,postinst,postrm}
    - change SNMP user from "snmp" to "Debian-snmp" to avoid confusion with
      non-system user. It satisfy to work with some LDAP system. Thanks to
      Vincent Bernat <bernat@debian.org> for suggestions.
      (Closes: #794641, #794647)
  * debian/snmp.preinst
    - revert "killall", it is unnecessary anymore (Closes: #781257)